lib/suspenders/app_builder.rb

```diff
@@ -170,7 +170,7 @@ def configure_action_mailer
       action_mailer_host "production", %{ENV.fetch("APPLICATION_HOST")}
       action_mailer_asset_host(
         "production",
-        %{ENV.fetch("ASSET_HOST", ENV.fetch("APPLICATION_HOST"))},
+        %q{"https://#{ENV.fetch("ASSET_HOST", ENV.fetch("APPLICATION_HOST"))}"}
```

Comment:
I think TLS in production is a reasonable assumption in 2020. I couldn't see anywhere else where we were explicitly specifying a protocol in the generated app code.